.github/workflows/ci.yml

```
@@ -93,6 +93,37 @@ jobs:
             fi
           # Each line of install_manifest.txt contains the location of an installed library or header
           done <install_manifest.txt
+          # Uninstall and reset build.
+          sudo xargs rm < install_manifest.txt || echo "transport_interface.h is installed twice because of duplicate copies in spoke repos."
+          cd .. && rm -rf build
+      - name: Configure CMake build for default install location and selective installation
+        run: |
+          curl https://cmake.org/files/v3.2/cmake-3.2.0-Linux-x86_64.tar.gz -o cmake.tar.gz
+          tar -xf cmake.tar.gz
```

Cmake is already installed on line 79
```suggestion
          curl https://cmake.org/files/v3.2/cmake-3.2.0-Linux-x86_64.tar.gz -o cmake.tar.gz
          tar -xf cmake.tar.gz
```